blockreassurance.php

```diff
@@ -19,339 +19,378 @@
 * needs please refer to http://www.prestashop.com for more information.
 *
 *  @author PrestaShop SA <contact@prestashop.com>
-*  @copyright  2007-2016 PrestaShop SA
-
+*  @copyright  2007-2019 PrestaShop SA
 *  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
 *  International Registered Trademark & Property of PrestaShop SA
 */
 
-if (!defined('_CAN_LOAD_FILES_')) {
+if (!defined('_PS_VERSION_')) {
     exit;
 }
 
 use PrestaShop\PrestaShop\Core\Module\WidgetInterface;
 
-include_once _PS_MODULE_DIR_.'blockreassurance/reassuranceClass.php';
+require_once _PS_MODULE_DIR_.'/blockreassurance/classes/ReassuranceActivity.php';
 
-class Blockreassurance extends Module implements WidgetInterface
+class blockreassurance extends Module implements WidgetInterface
 {
+    public $name;
+    public $version;
+    public $author;
+    public $need_instance;
+    public $module_key;
+    public $author_address;
+    public $controller_name;
+    public $bootstrap;
+    public $displayName;
+    public $description;
+    public $js_path;
+    public $css_path;
+    public $img_path;
+    public $img_path_perso;
+    public $lib_path;
+    public $docs_path;
+    public $logo_path;
+    public $module_path;
+    public $confirmUninstall;
+    public $ps_url;
+    public $folder_file_upload;
+
     private $templateFile;
 
     public function __construct()
     {
+        // Settings
         $this->name = 'blockreassurance';
+        $this->tab = 'seo';
+        $this->version = '4.0.0';
         $this->author = 'PrestaShop';
-        $this->version = '3.0.1';
-
+        $this->need_instance = 0;
+        $this->module_key = '938b96386d4d79aa7cb891439cb0ef11';
```

Comment:
Does it make sense, for a github module (not an Addons module) ?

Reply:
yes all module have a key